Postman Test Suite/Ed-Fi ODS-API Profile Test Suite.postman_collection.json

```diff
@@ -10779,7 +10791,7 @@
 										"method": "GET",
 										"header": [],
 										"url": {
-											"raw": "{{ApiBaseUrl}}/metadata/data/v3/resources/swagger.json",
+											"raw": "{{ApiBaseUrl}}/metadata/data/v3/resources/swagger.json?version=3",
```

Comment:
@mjaksn , @simpat-jesus does this mean version 3 is not the default?

Reply:
V3 is the default. I added this to indicate which version the updated Postman test expects, but it could be removed.